qiskit/ignis/verification/topological_codes/fitters.py

```diff
@@ -373,19 +395,158 @@ def matching(self, string):
 
         return logical_string
 
-    def get_logical_prob(self, results, algorithm='matching'):
+    def Nearest_Cluster(self, Cluster, graph, target):
```

Comment:
This won't pass the naming lint rules for method names, it should be `nearest_cluster`.

Reply:
Okay, yes. I will do that.

Reply:
Done